docs/docs-beta/docs/dagster-plus/access/rbac/audit-logs.md

```diff
@@ -4,4 +4,51 @@ displayed_sidebar: "dagsterPlus"
 sidebar_position: 4
 ---
 
-# Audit logs
\ No newline at end of file
+# Audit logs
+
+The Dagster Plus audit log enables enterprise organizations to track and attribute changes to their Dagster deployment.
```

Comment:
Should we use the word `enterprise` here? Perhaps it be something that aligns with our pricing plans, ex:

> The Dagster+ audit log enables Dagster+ Pro organizations...

Reply:
Also, maybe include a sentence or two about why having the ability to track and attribute deployment changes is helpful.